README.md

```diff
@@ -1 +1,3 @@
 https://twitchtokengenerator.com/ for generating your tokens is recommended.
+
+Currently works with 1.53.0-nightly

```

Comment:
Just fyi 1.53.0-nightly changes every day (nightly) nightly versions are usually referred to by the date ie nightly-2021-04-19